### DISCUSSION TITLE
Propuestas para adornar el mapa

### DISCUSSION THREAD

**Discussion by ma-robles** [grupoioa/gis_agm_series#27] [Ideas]
Escriban sus propuestas para agregar al mapa de manera que no se vea tan "plano" o tan vacío. @MaCuur como eres la experta en mapas yo creo que puedes tener alguna otra idea de que podemos poner (aparte de las isotermas).

**Comment by ma-robles** [comment]
Mi propuesta es poner la batimetría y puntos de interés. Entre los puntos de interés pueden estar plataformas y demás instalaciones en el GoM pero también algunas ciudades importantes o playas.

**Reply by MaCuur** [reply]
pues con la posición de los pozos petroleros creo que sería suficiente 
voy a pensarle en qué más hay de importancia 
jeje

**Reply by ma-robles** [reply]
que tal puertos y ciudades/poblaciones cercanas a la costa que pudieran ser afectadas en un evento?

**Reply by MaCuur** [reply]
pues sería toda la costa dependiendo del fenómeno meteorológico o de la afectación debido a las corrientes marinas

regularmente los pozos petroleros están asociados a las ciudades petroleras que al fin de cuentas son los principales focos económicos del golfo

No hace daño tal vez añadir esas =D... me late

**Reply by ma-robles** [reply]
@MaCuur creo que tú manejas arcgis o qgis o similares, ¿no? ¿que tanto trabajo te daría hacer un shape con esos puntos? Igual podemos incluir algún shape ya existente como primer versión, aunque tengan nombres de ciudades de todo el país (no solo de la costa).

**Reply by MaCuur** [reply]
Va, te lo mando al 🐁
